yaml-sort.js

```diff
@@ -88,28 +93,48 @@ argv.input.forEach((file) => {
     }
 
     const output =
-        argv.stdout || (argv.output === '.') || (isStdin && !argv.output)
-          ? process.stdout.fd
-          : (argv.output ? argv.output : file)
+      argv.stdout || (argv.output === '.') || (isStdin && !argv.output)
+        ? process.stdout.fd
+        : (argv.output ? argv.output : file)
 
     const content = fs.readFileSync(isStdin ? process.stdin.fd : file, argv.encoding)
 
-    const sorted = yaml.dump(yaml.load(content), {
+    const documents = yaml.loadAll(content)
+
+    if (!documents) {
```

Comment:
@Romain7495 Can't trigger this with a test. I think it's useless as it can't be triggered.

Reply:
I removed the condition, It was relevant only why my try catch, but removed as well